data/data/gcp/bootstrap/main.tf

```diff
@@ -21,7 +22,8 @@ data "ignition_config" "redirect" {
 }
 
 resource "google_compute_address" "bootstrap" {
-  name = "${var.cluster_id}-bootstrap-ip"
+  name   = "${var.cluster_id}-bootstrap-ip"
+  region = var.region
```

Comment:
So looking at the docs, https://www.terraform.io/docs/providers/google/r/compute_address.html#region

The region will be set to the region set in the google provider https://github.com/openshift/installer/blob/04c6785600adc5ce4ed84b15442546f0cb5ad8e8/data/data/gcp/main.tf#L11-L15

So I don't think we need to set the region here again...